torch/utils/_sympy/functions.py

```diff
@@ -1200,7 +1200,9 @@ def eval(cls, a):
                 return getattr(sympy, name)(a)
```

Comment:
I think we can lookup only once and cache in the closure for `getattr(math, name)` and `getattr(sympy, name)` to gain some performance.